CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.4
```

Comment:
This should probably be 1.1.0, since telling the compiler that `error` always throws is kind of a new feature.

Reply:
Done.